server/lib/url-utils.ts

```diff
@@ -4,9 +4,9 @@ import { types as CollectiveType } from '../constants/collectives';
 
 export const getEditRecurringContributionsUrl = collective => {
   if (collective.type === CollectiveType.USER) {
-    return `${config.host.website}/recurring-contributions`;
+    return `${config.host.website}/manage-contributions`;
   } else {
-    return `${config.host.website}/${collective.slug}/recurring-contributions`;
+    return `${config.host.website}/${collective.slug}/manage-contributions`;
```

Comment:
Seems unrelated to this PR?

Reply:
Yes. This was originally the backend for the manage-contributions branch, I'll update the title of the PR.